File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domene/Publiseringsdatoer.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene.ÅrstallOgKvartal;

@Data
@AllArgsConstructor
@Builder
public class Publiseringsdatoer {
  private String sistePubliseringsdato;
  private String nestePubliseringsdato;
  private ÅrstallOgKvartal gjeldendePeriode;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Publiseringsdatoer.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data

@Data
@AllArgsConstructor
@Builder
class Publiseringsdatoer {
    private val sistePubliseringsdato: String? = null
    private val nestePubliseringsdato: String? = null
    private val gjeldendePeriode: ÅrstallOgKvartal? = null
}
